app/components/App/styles.css

```diff
@@ -9,17 +9,30 @@ html, body {
   width: 100%;
 }
 
+a {
```

Comment:
I strongly do not recommend you to use general big css files, instead I suggest to use css-in-modules, the philosophy of load css when a component is mounted goes better (casa mejor) with React. Also you will save time on the first initial load.

If you have questions, let me know :)